src/ngx_http_lua_subrequest.c

```diff
@@ -1699,6 +1701,16 @@ ngx_http_lua_copy_request_headers(ngx_http_request_t *sr,
     part = &pr->headers_in.headers.part;
     header = part->elts;
 
+#if (NGX_HTTP_V3)
+    if (pr->headers_in.server.data != NULL) {
+        if (ngx_http_lua_set_input_header(sr, host_header,
+                                         pr->headers_in.server, 0) == NGX_ERROR)
```

Comment:
code style, parameters are not aligned

Reply:
This line hit the 80-character limit.

Reply:
It appears that the nginx core addresses this situation in the following way.
```suggestion
        if (ngx_http_lua_set_input_header(sr, host_header,
                                                                     pr->headers_in.server, 0)
             == NGX_ERROR)
```